docs/v7-migration.md

```diff
@@ -28,14 +28,14 @@ The "env" preset has been out for more than a year now, and completely replaces
 - `babel-preset-es2016`
 - `babel-preset-es2017`
 - `babel-preset-latest`
-- can combination of the above ^
+- A combination of the above ^
 
-We suggest using switching as those presets should be able to be substituted entirely with the env preset.
+These presets can be substituted with the "env" preset.
```

Comment:
It's more than "can", we are actively encouraging it.

Reply:
Yeah I guess it's a "should", don't think that's too opinionated because we deprecated the other ones